src/daemon/configuration.proto

```diff
@@ -35,10 +39,15 @@ message Configuration {
   }
 
   message Emitter {
-    required string socket_path = 1;
-    repeated Host remotes       = 2;
-    optional uint32 threads     = 3 [ default = 2 ];
-    optional bool only_failed   = 4 [ default = false ];
+    required string socket_path                = 1;
+    repeated Host remotes                      = 2;
+    optional uint32 threads                    = 3 [ default = 2 ];
+    optional bool only_failed                  = 4 [ default = false ];
+
+    optional uint32 poll_coordinators_interval = 5 [ default = 30 ];
```

Comment:
It's better to shorten the name (i.e. `poll_interval`) and add proper comment about coordinators.